packages/buildcop/src/buildcop.ts

```diff
@@ -510,7 +510,16 @@ buildcop.markIssueFlaky = async (
   const existingLabels = existingIssue.labels
     ?.map(l => l.name)
     .filter(l => !l.startsWith('buildcop'));
-  const labels = LABELS_FOR_FLAKY_ISSUE.concat(existingLabels);
+  let labels = LABELS_FOR_FLAKY_ISSUE;
+  // If existingLabels contains a priority: or type: label, don't add the
+  // default priority: and type: labels.
+  if (existingLabels?.find(l => l.startsWith('priority:'))) {
+    labels = labels.filter(l => !l.startsWith('priority:'));
```

Comment:
I am a little confused by this line.  Don't you want to `concat`?

Reply:
Changed the variable name to hopefully make this more clear. Please take another look.